packages/node/README.md

```diff
@@ -15,31 +15,49 @@ npm install readmeio
 
 ## Usage
 
-Just add the middleware to express, and that's it!
+Just add the middleware to [Express](https://expressjs.com/), and that's it!
 
 ```javascript
 const readme = require('readmeio');
 
 app.use(readme.metrics('<<apiKey>>', req => ({
-  id: req.project._id,
-  label: req.project.name,
-  email: req.project.owner
+  id: req.<userId>,
+  label: req.<userNameToShowInDashboard>,
+  email: req.<userEmailAddress>,
 })));
 ```
 
 View full documentation here: [https://docs.readme.com/docs/sending-logs-to-readme-with-nodejs](https://docs.readme.com/docs/sending-logs-to-readme-with-nodejs)
 
+### Configuration Options
+There are a few options you can pass in to change how the logs are sent to ReadMe. These are passed in an object as the third parameter to the `readme.metrics`.
+
+```js
+const readme = require('readmeio');
+const env = process.env.NODE_ENV;
+
+app.use(readme.metrics('', req => ({
+  id: req.<userId>,
+  label: req.<userNameToShowInDashboard>,
+  email: req.<userEmailAddress>,
+}), {
+  development: env !== 'production',
+  bufferLength: 1,
+  blacklist: [[arrayOfSensitiveKeysToOmit]],
+  whitelist: [[arrayofKeysOnlyToSend]],
+}));
+```
+
+| Option | Use |
+| :--- | :--- |
+| development | **default: false** If true, the log will be separate from normal production logs. This is great for separating staging or test data from data coming from customers |
+| bufferLength | **default: 10** By default, we only send logs to ReadMe after 10 requests are made. Depending on the usage of your API it make make sense to send logs more or less frequently |
+| blacklist | **optional** An array of keys from your API requests and responses headers and bodies that you wish to blacklist from sending to ReadMe.<br /><br />If you configure a blacklist, it will override any whitelist configuration. |
+| whitelist | **optional** An array of keys from your API requests and responses headers and bodies that you only wish to send to ReadMe. |
+| baseLogUrl | **optional** This is the base URL for your ReadMe project. Normally this would be `https://projectName.readme.io` or `https://docs.yourdomain.com`, however if this value is not supplied, a request to the ReadMe API will be made once a day to retrieve. This data will be cached to `node_modules/.cache/readmeio`.
```

Comment:
Will add this to https://docs.readme.com/metrics/docs/sending-logs-to-readme-with-nodejs when this hits NPM.